package.json

```diff
@@ -148,6 +148,7 @@
     "md5-file": "^5.0.0",
     "npm-watch": "^0.10.0",
     "prop-types": "^15.7.2",
+    "react-error-overlay": "^6.0.9",
```

Comment:
This doesn't seem to be used as part of this commit...

Reply:
It's mentioned in the comment...I need it to fix my react debugging issues.